website/apps/homepage/views.py

```diff
@@ -25,6 +33,12 @@ def get_quote(self):
             )
 
     def get_context_data(self):
+        if self.request.mission.memorial:
+            people, more_people = mission_people(self.request)
```

Comment:
It took me a minute to work out what `people` and `more_people` meant. Could we rename these variables to be more descriptive? `astronauts` and `mission_ops_people` maybe?